src/privacy_review_assistant_group.py

```diff
@@ -35,7 +38,7 @@ def terminate_chat(message):
             return message_sender_name != user_agent.name
         assistant = AssistantAgent(
             name="Threat_Model_Evaluator",
-            description="An agent that manages a group chat for threat modeling validation and evaluation.",
+            description="An agent that manages a group chat for threat modeling validation and evaluation but should never be used the user does requests XML validation, ONLY when the user requests image validateion.",
```

Comment:
Don't think you need this right? They're mutually exclusive?

Reply:
Meaning because I have system message I don't need description? My understanding was that system message was for the prompt execution to get a result and description was for the group chat but correct me if I'm wrong.

Reply:
No i meant the "but should never be used the user does requests XML validation, ONLY when the user requests image validateion." part. 
This assistant only comes into play if XML assistant is not being used right?

Reply:
Ah yes, I should remove this. The goal of this was to test out being able to test out being able to use different validation methods based on the request, but that didn't seem to work too well.